app/javascript/src/styles/layout/_devise_errors.scss

```diff
@@ -40,3 +40,36 @@
     }
   }
 }
+
+/*flash*/
+.alert-error {
```

Comment:
Can we use bootstrap alerts message style? https://getbootstrap.com/docs/4.0/components/alerts/